### ISSUE TITLE
category_test

### ISSUE THREAD

**Issue by smaliaquib** [aakashjhawar/dress-pattern-recognition-using-CNN#2]
category test code is missing

**Comment by sravan8074** [comment]
He misled us. There is no way that this model scored ~98% accuracy. And the other thing he mentioned binary cross-entropy in the loss function which is an error because it belongs to multi-class classification.
